Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -935,6 +950,16 @@ def user_reactivate_command(client, args):
     )
 
 
+def download_file_command(client, args):
+    file_hash = args.get("hash")
+    file_chunks = []
+    response = client.download_file(file_hash)
+    for chunk in response.iter_content(chunk_size=128):
+        if chunk:
+            file_chunks.append(chunk)
+    return fileResult(file_hash, data=b"".join(file_chunks))
```

Comment:
Does this mean XSOAR will have the file stored with the hash as the filename? If the XSOAR user wants to download it to their machine do they then get it with the hash as the name too?

Reply:
Yes, that is exactly what happens. What do you think about that?